compiler/src/dotty/tools/dotc/typer/Namer.scala

```diff
@@ -54,6 +54,7 @@ class Namer { typer: Typer =>
   val TypedAhead      : Property.Key[tpd.Tree]            = new Property.Key
   val ExpandedTree    : Property.Key[untpd.Tree]          = new Property.Key
   val ExportForwarders: Property.Key[List[tpd.MemberDef]] = new Property.Key
+  val WildcardExport  : Property.Key[Unit]                = new Property.Key
```

Comment:
IMO, attachements are a very hacky solution to any problem and we shouldn't use them unless absolutely necessary. We need to preserve export trees until Pickler for IDEs/tooling anyway, so why would the attachment be needed?

Reply:
the trees are dropped at typer, so i wanted to try this first instead of having to handle a new tree node everywhere else

Reply:
Secondly, I am unsure how downstream code should interpret an export statement alongside the forwarder methods that were generated for it

Reply:
should exports appear in tasty reflect?

Reply:
Same as import statements: you can ignore them if you don't do anything with them, but e.g. scala3doc might be interested in them.

Reply:
so we need to update tasty